File: src/main/java/classe/GraphJourSemaine.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package classe;

/**
 *
 * @author tolot
 */
public class GraphJourSemaine {
    int lundi;
    int mardi;
    int mercredi;
    int jeudi;
    int vendredi;
    int samedi;

    public GraphJourSemaine() {
    }
    int dimanche;

    public int getLundi() {
        return lundi;
    }

    public void setLundi(int lundi) {
        this.lundi = lundi;
    }

    public int getMardi() {
        return mardi;
    }

    public void setMardi(int mardi) {
        this.mardi = mardi;
    }

    public int getMercredi() {
        return mercredi;
    }

    public void setMercredi(int mercredi) {
        this.mercredi = mercredi;
    }

    public int getJeudi() {
        return jeudi;
    }

    public void setJeudi(int jeudi) {
        this.jeudi = jeudi;
    }

    public int getVendredi() {
        return vendredi;
    }

    public void setVendredi(int vendredi) {
        this.vendredi = vendredi;
    }

    public int getSamedi() {
        return samedi;
    }

    public void setSamedi(int samedi) {
        this.samedi = samedi;
    }

    public int getDimanche() {
        return dimanche;
    }

    public void setDimanche(int dimanche) {
        this.dimanche = dimanche;
    }

    public GraphJourSemaine(int lundi, int mardi, int mercredi, int jeudi, int vendredi, int samedi, int dimanche) {
        this.lundi = lundi;
        this.mardi = mardi;
        this.mercredi = mercredi;
        this.jeudi = jeudi;
        this.vendredi = vendredi;
        this.samedi = samedi;
        this.dimanche = dimanche;
    }
    
    
}

File: Dashboard/GraphJourSemaine.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Dashboard
{
    public class GraphJourSemaine
    {
        int lundi;
        int mardi;
        int mercredi;
        int jeudi;
        int vendredi;
        int samedi;
        int dimanche;

        public GraphJourSemaine(int lundi, int mardi, int mercredi, int jeudi, int vendredi, int samedi, int dimanche)
        {
            this.lundi = lundi;
            this.mardi = mardi;
            this.mercredi = mercredi;
            this.jeudi = jeudi;
            this.vendredi = vendredi;
            this.samedi = samedi;
            this.dimanche = dimanche;
        }

        public int Lundi { get => lundi; set => lundi = value; }
        public int Mardi { get => mardi; set => mardi = value; }
        public int Mercredi { get => mercredi; set => mercredi = value; }
        public int Jeudi { get => jeudi; set => jeudi = value; }
        public int Vendredi { get => vendredi; set => vendredi = value; }
        public int Samedi { get => samedi; set => samedi = value; }
        public int Dimanche { get => dimanche; set => dimanche = value; }
    }
}
